liberty-maven-plugin/src/it/dev-it/src/test/java/net/wasdev/wlp/test/dev/it/BaseDevTest.java

```diff
@@ -245,6 +249,7 @@ protected static void cleanUpAfterClass(boolean isDevMode, boolean checkForShutd
 
       if (logFile != null && logFile.exists()) {
          assertTrue(logFile.delete());
+         assertTrue(logErrorFile.delete());
```

Comment:
What if this file does not exist? Then this line will fail. Check for its existence separately.

Reply:
You're right, I missed that. Pushed a correction. For now, I'll wait what your changes in #1642 will bring.